proof/invariant-abstract/SchedContext_AI.thy

```diff
@@ -814,13 +814,22 @@ lemma commit_times_invs_helper:
                   wp: valid_irq_node_typ hoare_vcg_imp_lift')
   done
 
+lemma get_sc_active_sp:
+  "\<lbrace>P\<rbrace>
+   get_sc_active sc_ptr
+   \<lbrace>\<lambda>rv s. P s
+           \<and> (\<exists>sc n. ko_at (kernel_object.SchedContext sc n) sc_ptr s \<and> rv = (0 < sc_refill_max sc))\<rbrace>"
```

Comment:
Do we need the `kernel_object_.` qualifier here? I would have thought that at this point in the theories there is only one `SchedContext` constructor defined.

Reply:
Just saw that it was moved from Refine, where the qualifier would have been needed. But in AInvs we should be able to drop it.